WorkloadManagementSystem/Service/PilotsLoggingHandler.py

```diff
@@ -14,33 +14,34 @@
 from DIRAC.Resources.MessageQueue.MQCommunication import createConsumer
 
 from DIRAC.Core.DISET.RequestHandler import RequestHandler
-from DIRAC import S_OK
-
+from DIRAC import S_OK, gConfig
 
 class PilotsLoggingHandler( RequestHandler ):
   """Server side functions for Pilots Logging service"""
 
+  consumerSet = None
+  pilotsLoggingDB = None
+
   @classmethod
-  def initializeHandler( cls, serviceInfo ):
+  def initializeHandler( cls, serviceInfoDict ):
     """Initialization of Pilots Logging service
     """
     cls.consumersSet = set()
     cls.pilotsLoggingDB = PilotsLoggingDB()
-
-    result = createConsumer( cls.srv_getCSOption("PilotsLoggingQueue"), callback = cls.consumingCallback )
+    queue = cls.srv_getCSOption("PilotsLoggingQueue")
+    # This is pretty awful hack. Somehow, for uknown reason, I cannot access CS with srv_getCSOption.
+    # The only way is using full CS path, so I'm using it as a backup solution.
```

Comment:
Put it in `initialize` method.

Reply:
`initialize` is called on each call to the service. Here I want to create consumer on service startup.